app/controllers/concerns/api/v2/parameter_deprecation.rb

```diff
@@ -0,0 +1,61 @@
+# This module adds a pre-action filter that will handle deprecated properties.
```

Comment:
I was not aware of that deprecation at all. Now that we already have more than one deprecation, we definitely should DRY it up. IMHO it will give us a single place to manage such things, which will ease up finding these places later on.
I am not sure that the module introduces that much complicity as opposed to dedicated method deprecation.

I think that uniformity of the handling takes place here. A good example is the deprecation warning - in my previous version, I was sending a warning message, but @orrabin asked me to remove the warning.